bindings/java/src/org/sleuthkit/datamodel/OsAccountRealmManager.java

```diff
@@ -63,103 +61,126 @@
 	 *
 	 * @throws TskCoreException If there is an error creating the realm.
 	 */
+	
+	// RAMAN TBD: does this go away ??? Confirm with Brian.
 	public OsAccountRealm createRealmByName(String realmName, Host host) throws TskCoreException {
```

Comment:
Yes, this can at least be not public or go away if we do not need it internally.